session.go

```diff
@@ -141,18 +176,20 @@ func ConfigurePings(interval time.Duration, timeout time.Duration) {
 
 func initSession(dc *DeviceContext, useMQTT bool) error {
 	sess = &session{
-		dc:        dc,
-		usingMQTT: useMQTT,
-		cmdQueue:  make(chan *DeviceCommand, commandQueueLength),
-		evtQueue:  make(chan *DeviceEvent, eventQueueLength),
+		dc:          dc,
+		usingMQTT:   useMQTT,
+		cmdQueue:    make(chan *DeviceCommand, commandQueueLength),
+		evtQueue:    make(chan *DeviceEvent, eventQueueLength),
+		sendKvQueue: make(chan *KeyValue, eventQueueLength),
+		recvKvQueue: make(chan *KeyValue, eventQueueLength),
```

Comment:
do you have any name suggestion?

I named the queue with send/recv depending on the data direction